ext/Cargo.toml

```diff
@@ -16,20 +16,31 @@ ruby-api = []
 
 [dependencies]
 lazy_static = "1.4.0"
-magnus = { version = "0.5.3", features = ["rb-sys-interop"] }
-rb-sys = "~0.9.81"
-wasmtime = { version = "= 9.0.1" }
-wasmtime-wasi = "= 9.0.1"
-wasi-common = "= 9.0.1"
-wasi-cap-std-sync = "= 9.0.1"
+# For ruby-head support, we have to lock to 0.5.4 until
```

Comment:
Should this comment be updated to say `0.5.5`?

Reply:
Fixed